bin/source

```diff
@@ -31,7 +31,17 @@ apt_src() (
 	trap 'cd / && rm -rf "$tmp_dir"' EXIT
 	cd "$tmp_dir"
 
-	apt-get source --only-source --download-only "$1"
+	# if the src repo provides multiple sources for the same package
+	# download the src with the same version as the candidate
+	if [[ "$1" = *=* ]]; then
```

Comment:
this way to call `apt_src` is new to me. are we doing this anywhere?

Reply:
can we document the expected/possible parameters for this function?

what I understand

- optional parameter `--ignore-orig`
- possible parameter pkg=version like `vim=9.0.42`
- possible parameter pkg like `vim`

is this correct?

Reply:
> this way to call apt_src is new to me. are we doing this anywhere?

[Not yet](https://github.com/search?q=org%3Agardenlinux+apt_src&type=code)

> can we document the expected/possible parameters for this function?

👍🏻 , for now I suggest to add comments above this function. 
We need to document the package build helper at some point properly.

Reply:
Added commit with simple docs to this PR

Reply:
thanks!